### PR TITLE
Stateless text embedder

### PR DESCRIPTION
**Pull request by olgavrou**
and general cleanup around it

### CODE REVIEW

File: vw_example_builder.py [hunk 1]
```diff
@@ -56,23 +101,13 @@ def embed_context(self, context: Any):
             - If context is provided as a dictionary, then it should be a single dictionary where the keys are namespace names and the values are the corresponding strings of the context (e.g. {"namespace1": "part of context", "namespace2": "another part of the context"})
         """
         if isinstance(context, str):
```
Comment (ataymano) [inline vw_example_builder.py]: do we still need this if?

File: vw_example_builder.py [hunk 2]
```diff
@@ -32,19 +87,9 @@ def embed_actions(self, actions: List):
             - If actions are provided as a list of dictionaries, each action should be a dictionary where the keys are namespace names and the values are the corresponding action strings (e.g. actions = [{"namespace1": "action1", "namespace2": "action2"}, {"namespace1": "action3", "namespace2": "action4"}])
         """
         if actions and isinstance(actions[0], str):
```
Comment (ataymano) [inline vw_example_builder.py]: do we still need this if?

Comment (olgavrou) [reply vw_example_builder.py]: if we move the default namespace creation to the cb text embedder then I guess we don't need to check

Comment (olgavrou) [reply vw_example_builder.py]: and then we don't need the default namespace if we assume that everything comes in with a namespace

Comment (ataymano) [reply vw_example_builder.py]: isn't just the whole function collapsable to embed(actions, "Actions") already?

Comment (olgavrou) [reply vw_example_builder.py]: ah yeah ok done

**Pull request merged**